packages/neos-ui/src/Containers/RightSideBar/Inspector/index.js

```diff
@@ -73,17 +73,27 @@ export default class Inspector extends PureComponent {
 
     state = {
         secondaryInspectorComponent: null,
-        toggledPanels: {}
+        toggledPanels: {},
+        viewConfiguration: null,
+        originalViewConfiguration: null
     };
 
     constructor(props) {
         super(props);
-        this.cloneViewConfiguration(props);
+        if (props.focusedNode) {
+            this.state.viewConfiguration = props.nodeTypesRegistry.getInspectorViewConfigurationFor($get('nodeType', props.focusedNode));
+            this.state.originalViewConfiguration = {...this.state.viewConfiguration};
+        }
     }
 
     componentWillReceiveProps(newProps) {
         if (newProps.focusedNode !== this.props.focusedNode) {
-            this.cloneViewConfiguration(newProps);
+            this.setState(prevState => ({
+                viewConfiguration: newProps.nodeTypesRegistry.getInspectorViewConfigurationFor($get('nodeType', newProps.focusedNode)),
+                originalViewConfiguration: {...prevState.viewConfiguration}
+            },
+            () => this.preprocessViewConfigurationDebounced()
```

Comment:
why is this call necessary? We already have one in `componentDidUpdate`, right?

Reply:
Without this callback the view configuration processing was off (for example a conditional field in the inspector would vanish after applying changes to the node). Of course this should not be necessary since we do the processing after any update and trigger a re-rendering. But I think this bug is connected to the general current update issue.